rfcs/README.md

```diff
@@ -0,0 +1,98 @@
+However, if the issue or discussion results in an idea for a significant 
+change or new feature that affects the library's public API or architecture, 
+we recommend creating a formal document that provides
+a detailed description and design of the proposed feature.
+
+This directory contains design documents (RFCs) approved, 
+implemented, or rejected for implementation.
+
+## RFC Directory Structure
+
+The design documents are stored in the `rfcs` directory, each placed 
+in a subdirectory under an `rfcs/<state>` directory. The possible states are:
+
+1. Proposed
+2. Experimental
+3. Supported
+4. Archived
+
+The `rfcs/proposed` directory contains RFCs for approved proposals
+that yet need to be implemented. These documents describe the overall design
+and API for the proposed functionality.
+
+The `rfcs/experimental` directory contains RFCs for experimental library features.
+In addition to the design, these documents describe the criteria for the described
+functionality to exit the experimental status.
+
+The `rfcs/supported` directory contains documents for the fully supported features,
+both implemented according to the library specification and provided as extensions.
+
+The `rfcs/archived` directory contains rejected proposals and documents for
+the former functionality that has been removed.
+
+A subdirectory for an RFC should have a name of the form `<library_feature>_<extension_description>`
+and should contain a `README.md` file that either is the RFC document
+or links to other files and Web resources that describe the functionality.
+The directory can contain other supporting files such as images or formulas,
+as well as sub-proposals / sub-RFCs.
+
+## General Process
+
+You can collect initial feedback on an idea and input for a formal RFC proposal
+using a GitHub discussion. Add the "RFC" label to the discussion to indicate
+the intent. The discussion can also be used to reference relevant information
+and keep track of the progress.
+
+To create a new RFC document, open a pull request (PR) to add it to the `rfcs/proposed` directory.
+A template for new RFCs is available as [template.md](template.md).
+Use it to create the `README.md` file in a subdirectory of `rfcs/proposed` named
+`<library_feature>_<extension_description>`. For example,
+a proposal for adding a bitonic sorting algorithm working with C++ ranges would be put
+into the `rfcs/proposed/range_algorithms_bitonic_sort` directory.
+Put other files referenced by the `README.md` file, such as figures, into the same directory.
+The "RFC" label can be used to mark PRs containing RFC/design proposals.
+
+The RFC approval process generally follows the guidelines in the [UXL Foundation Operational Procedures](
+https://github.com/uxlfoundation/uxl_operational_procedures/blob/release/Process_Documents/Organization_Operational_Process.md#review--approval-process).
+Once at least two maintainers approve the PR, it is merged into the main branch
```

Comment:
Please add commas:
Once**,** at least**,** two maintainers approve the PR, it is merged into the main branch
as an RFC proposed for implementation.